packages/plugins/rubygem-openscap/rubygem-openscap.spec

```diff
@@ -1,66 +1,40 @@
+# template: nonscl
 %global gem_name openscap
 
 Name: rubygem-%{gem_name}
-Version: 0.4.7
+Version: 0.4.9
 Release: 1%{?dist}
 Summary: A FFI wrapper around the OpenSCAP library
 Group: Development/Languages
-License: GPLv2+
+License: GPL-2.0
 URL: https://github.com/OpenSCAP/ruby-openscap
 Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
 
+# start specfile generated dependencies
+Requires: ruby(release)
+Requires: ruby
 Requires: ruby(rubygems)
 Requires: rubygem(ffi) >= 1.0.9
-# require libopenscap.so.8 in an arch neutral way
-Requires: openscap >= 1.2.9
```

Comment:
I suspect you need to add this below the generated dependencies to make sure it's present.